deltachat-ios/Controller/ChatListController.swift

```diff
@@ -241,11 +248,14 @@ class ChatListController: UITableViewController {
         tableView.rowHeight = ContactCell.cellHeight
     }
 
-    
+    private var isInitial = true
     @objc func applicationDidBecomeActive(_ notification: NSNotification) {
         if navigationController?.visibleViewController == self {
-            startTimer()
-            refreshInBg()
+            if !isInitial {
+                startTimer()
+                refreshInBg()
+                isInitial = false
```

Comment:
as a general precaution, it probably makes sense to set `isInitial = false` two lines further up:

```suggestion
                isInitial = false
                startTimer()
                refreshInBg()
```